### PR TITLE
Hook up IdCards page with actual data

### PR DESCRIPTION
**Pull request by limemloh**
## Purpose

Display actual identities in ID cards page

## Changes

- Factor out `useEditableValue` for shared logic between account name and identity name editing.

### CODE REVIEW

File: packages/browser-wallet/src/popup/popupX/shared/IdCard/IdCard.tsx [hunk 1]
```diff
@@ -2,35 +2,64 @@ import React, { ReactNode } from 'react';
...
-                {onEditName && <Button.Secondary label="Edit Name" />}
+                <Text.Main>{editable.value}</Text.Main>
+                {editable.isEditing ? (
+                    <>
+                        <Button.Secondary label="Save" onClick={editable.onComplete} />
+                        <Button.Secondary label="Abort" onClick={editable.onAbort} />
+                    </>
+                ) : (
+                    onNewName && <Button.Secondary label="Edit Name" onClick={editable.onEdit} />
+                )}
```
Comment (soerenbf) [inline packages/browser-wallet/src/popup/popupX/shared/IdCard/IdCard.tsx:46]: It itches in an annoying place in my brain when the buttons to edit and save/abort are not the same as for the account.. But I guess this is what the design specifies 🤷

File: packages/browser-wallet/src/popup/popupX/pages/Restore/RestoreResult.tsx [hunk 2]
```diff
@@ -29,7 +30,7 @@ export default function RestoreResult() {
             <Page.Top heading={t('result')} />
             <Page.Main>
                 <Text.Capture>{t('recoveredIds')}</Text.Capture>
-                <IdCard rowsIdInfo={rowsIdInfo} />
+                <IdCard rowsIdInfo={rowsIdInfo} idProviderName="TODO" identityName="TODO" />
```
Comment (soerenbf) [inline packages/browser-wallet/src/popup/popupX/pages/Restore/RestoreResult.tsx:33]: These are out of scope or? Can we just add them now?

File: packages/browser-wallet/src/popup/popupX/pages/Onboarding/IdSubmitted.tsx [hunk 3]
```diff
@@ -20,7 +21,7 @@ export default function IdSubmitted() {
             <Page.Top heading={t('yourId')} />
             <Page.Main>
                 <Text.Capture>{t('idSubmitInfo')}</Text.Capture>
-                <IdCard rowsIdInfo={rowsIdInfo} />
+                <IdCard rowsIdInfo={rowsIdInfo} idProviderName="TODO" identityName="TODO" />
```
Comment (soerenbf) [inline packages/browser-wallet/src/popup/popupX/pages/Onboarding/IdSubmitted.tsx:24]: And here

File: packages/browser-wallet/src/popup/popupX/shared/IdCard/IdCard.tsx (hunk 1 above)
Comment (soerenbf) [inline packages/browser-wallet/src/popup/popupX/shared/IdCard/IdCard.tsx:46]: These button labels should be in the `i18n/en.ts` file

**Review by soerenbf — COMMENTED** [review]
When I attempt to edit the name, it seems to not be saved? Is this out of scope of this?

File: packages/browser-wallet/src/popup/popupX/shared/IdCard/IdCard.tsx (hunk 1 above)
Comment (limemloh) [reply packages/browser-wallet/src/popup/popupX/shared/IdCard/IdCard.tsx:46]: The design only contains the `Edit name` button, the `Save` and `Abort` are me improvising.
But I agree, it is not consistent.

File: packages/browser-wallet/src/popup/popupX/pages/Onboarding/IdSubmitted.tsx (hunk 3 above)
Comment (limemloh) [reply packages/browser-wallet/src/popup/popupX/pages/Onboarding/IdSubmitted.tsx:24]: We need to hook up the identity issuance first

File: packages/browser-wallet/src/popup/popupX/shared/IdCard/IdCard.tsx (hunk 1 above)
Comment (soerenbf) [reply packages/browser-wallet/src/popup/popupX/shared/IdCard/IdCard.tsx]: Yeah.. let's just keep this then.. It's a minor thing anyway

**Review by soerenbf — APPROVED** [review]

**Review by Ivan-Mahda — APPROVED** [review]
